### PR TITLE
Add a steam import service

### PR DESCRIPTION
**Pull request by AnthonySuper**
This uses vastly fewer SQL calls, which is good.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -84,6 +84,12 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
 
+  gem 'irb'
```
Comment (connorshea) [inline Gemfile:87]:
```suggestion
```

**Comment by connorshea** [comment]
TODO (for me): Make sure this lets you import your games more than once without erroring.

File: app/services/steam_import_service.rb [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+          game_id: stored_id_hash[game_info['appid']],
+          user_id: user.id,
+          created_at: create_time,
+          updated_at: create_time
+        }
+      end
+    attrs = attrs.to_a
+    added_purchases =
+      if attrs.any?
+        inserted = GamePurchase.insert_all(attrs, unique_by: [:game_id, :user_id])
```
Comment (connorshea) [inline app/services/steam_import_service.rb:59]: this doesn't perform validations and that scares me a little

But more importantly, this doesn't perform callbacks >:(

Comment (AnthonySuper) [reply app/services/steam_import_service.rb:59]: You already disable the activity stuff by disabling bullet.

It's this or the code is slow AF okay

Comment (connorshea) [reply app/services/steam_import_service.rb:59]: Disabling bullet doesn't do anything to validations/callbacks afaik?

Comment (AnthonySuper) [reply app/services/steam_import_service.rb:59]: Oh right. I can make this `.find_or_create_by`, which is still *extremely* slow compared to a big upsert, but it'll still be faster than the current scheme.

**Pull request merged**